### ISSUE TITLE
Resizing texture causes other textures to resize

### ISSUE THREAD

**Issue by pvcraven** [pvcraven/arcade#506]
*From reddit:*

After upgrading to the latest version of arcade with pip(appears to be v2.0.9) loading the same image multiple times using load_texture gives a reference to the same texture object, making any changes to size affect all copies.

ex:
``` python
import arcade

SCREEN_WIDTH = 400
SCREEN_HEIGHT = 260
SCREEN_CENTER_X = SCREEN_WIDTH*0.5
TOP_SPRITE_Y = SCREEN_HEIGHT*0.75
BOTTOM_SPRITE_Y = SCREEN_HEIGHT*0.25
FILE_NAME = "GraphicsFiles/sprites/warrior_south_0.png"

class Master(arcade.Window):
    def __init__(self):
        arcade.Window.__init__(self, SCREEN_WIDTH,  SCREEN_HEIGHT)
        self.texture_1 = arcade.load_texture(FILE_NAME)
        self.texture_2 = arcade.load_texture(FILE_NAME)
    def run(self):
        arcade.run()
    def on_draw(self):
        arcade.start_render()
        arcade.draw_texture_rectangle(SCREEN_CENTER_X, TOP_SPRITE_Y, self.texture_1.width, self.texture_1.height, self.texture_1)
        arcade.draw_texture_rectangle(SCREEN_CENTER_X, BOTTOM_SPRITE_Y, self.texture_2.width, self.texture_2.height, self.texture_2)
        arcade.finish_render()

    def on_mouse_release(self, x, y, button, modifiers):
        self.texture_2.height*=1.5


if __name__ == '__main__':
    app = Master()
    app.run()
```
​
a mouse click will change the height of both textures, not just texture_2.

I know I can work around this with copy statements, but this behavior seems counterintuitive, and my code didn't have this problem with earlier versions. leading me to think this is unintended behavior. or am I just missing something?

**Comment by TheCodeSummoner** [comment]
That's because your `load_texture.texture_cache` in `draw_commands.py` is a `dict`, which is then used in `load_texture` closure. Note that by uncommenting the following lines (312, 313):

```python
if cache_name in load_texture.texture_cache:
    return load_texture.texture_cache[cache_name]
```

a mouse click will no longer change the height of both textures. The problem described occurred because both textures were under the same key (png file path). You should instead store textures under more unique keys - such as a hash or their built-in representation.

You can also create a shallow copy by using `copy` from the `copy` module:

```python
if cache_name in load_texture.texture_cache:
    return copy(load_texture.texture_cache[cache_name])
```

I don't think there is a better way to handle this situation, mostly because logically thinking these 2 textures are in fact separate, therefore you can't treat them with the same reference. I am happy to create a pull request if any of the solutions is to your liking.

**Comment by pvcraven** [comment]
Still an issue. Updated code:
```
import arcade

SCREEN_WIDTH = 400
SCREEN_HEIGHT = 260
SCREEN_CENTER_X = SCREEN_WIDTH*0.5
TOP_SPRITE_Y = SCREEN_HEIGHT*0.75
BOTTOM_SPRITE_Y = SCREEN_HEIGHT*0.25
FILE_NAME = ":resources:images/animated_characters/female_person/femalePerson_idle.png"

class Master(arcade.Window):
    def __init__(self):
        arcade.Window.__init__(self, SCREEN_WIDTH,  SCREEN_HEIGHT)
        self.texture_1 = arcade.load_texture(FILE_NAME)
        self.texture_2 = arcade.load_texture(FILE_NAME)
    def run(self):
        arcade.run()
    def on_draw(self):
        arcade.start_render()
        arcade.draw_texture_rectangle(SCREEN_CENTER_X, TOP_SPRITE_Y, self.texture_1.width, self.texture_1.height, self.texture_1)
        arcade.draw_texture_rectangle(SCREEN_CENTER_X, BOTTOM_SPRITE_Y, self.texture_2.width, self.texture_2.height, self.texture_2)

    def on_mouse_release(self, x, y, button, modifiers):
        self.texture_2.height*=1.5


if __name__ == '__main__':
    app = Master()
    app.run()
```

**Comment by pvcraven** [comment]
Fixed with version 2.2.8.
